gateway/1.5/modules/ROOT/pages/policies-pdk-debug-local.adoc

```diff
@@ -146,3 +145,17 @@ curl --location --request POST 'http://0.0.0.0:8081/some/route' \
 ----
 
 The backend service returns an echo response including the requests’s route, body, and headers sent with any modification performed by the custom policy or additional policies. Any logs included in the custom policy's source code are visible in the terminal running the containers.
+
+== Loglevel
+
+By default, the PDK debugging environment enables logs with the `debug` loglevel configured.
+
+To change the loglevel, edit the `logging.runtimeLogs.logLevel` value in `test/config/logging.yaml`.
+
+To learn more about editing a Flex Gateway's logging configuration file, see xref:flex-local-third-party-logs-config.adoc[].
+
+== Use the Rust Debugger
+
+Visual Studio Code (VS Code) includes a Rust debugger that can run unit tests. You cannot use the VS Code Debugger while the policy is deployed on a Flex Gateway. Meaning that to create the API mocking, you must use unit tests for the specific logic you need to debug.
```

Comment:
Nitpick: "Meaning" in this context might not translate properly. I don't think you necessarily need a transition between those two sentences.